service/src/main/java/org/cbioportal/service/impl/GenericAssayServiceImpl.java

```diff
@@ -80,15 +80,15 @@
                 }
             } else {
                 List<GenericAssayAdditionalProperty> additionalProperties = genericAssayRepository.getGenericAssayAdditionalproperties(distinctStableIds);
+                Map<String, List<GenericAssayAdditionalProperty>> additionalPropertiesGroupedByStableId =
+                    additionalProperties.stream().collect(Collectors.groupingBy(GenericAssayAdditionalProperty::getStableId));
```

Comment:
`additionalProperties` is only being used in the next line, so you might as well clean this up a bit:
```suggestion
                Map<String, List<GenericAssayAdditionalProperty>> additionalPropertiesGroupedByStableId =
                    genericAssayRepository.getGenericAssayAdditionalproperties(distinctStableIds).stream()
                        .collect(Collectors.groupingBy(GenericAssayAdditionalProperty::getStableId));```